wordpress/wp-content/plugins/memberful-wp/stylesheets/admin.css

```
@@ -167,6 +167,45 @@ Global Marketing Tools
   padding-left: 2em;
 }
 
+/*---------------------------------------------------------
```

Stylelint (17.12.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)